parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -239,7 +239,7 @@ installContainerdWasmShims(){
             shims_to_download+=("wws")
         fi
         containerd_wasm_url=$(evalPackageDownloadURL ${PACKAGE_DOWNLOAD_URL})
-        downloadContainerdWasmShims $download_location $containerd_wasm_url "v$version" "${shims_to_download[@]}" # adding v to version for simplicity
+        downloadContainerdWasmShims $download_location $containerd_wasm_url "$version" "${shims_to_download[@]}"
```

Comment:
is this related to shellcheck ?  is $version now storing the `v` ?

Reply:
Let me add the `v` back and do a test now

Reply:
I will keep the 'v' for now, though I found there is a bug for that. But I wanted to keep this PR pure for shellcheck.